src/manual/endpoint-kubernetes.adoc

```diff
@@ -69,6 +69,8 @@ In case you need to set the client configuration explicitly on your environment
               object-mapper="objectMapper"/>
 ----
 
+*Note*: You have the option to set an `oauthToken` for the client instead of using `username` and `password`.
```

Comment:
I wonder if we could check if it is **either** the token **or** username and password.. a check would probably make sense; [here](https://github.com/citrusframework/citrus/blob/deac0ef52e2735ac46ce6dcc9753f34d8466a77a/connectors/citrus-kubernetes/src/main/java/org/citrusframework/kubernetes/client/KubernetesClientBuilder.java#L41).

Reply:
check was added to KubernetesClientConfigParser